lib/proxy.js

```diff
@@ -301,24 +301,40 @@ function makeWsProxy(host, port, urlPrefix, isHttps) {
   };
 };
 
-function checkProxiedHost(host, port) {
+function checkProxiedHost(host, port, userConfig) {
   const client = new net.Socket();
+  const reconnectDelay = 5000;
```

Comment:
Should this be a const inside or outside the function? Usually, values that we want to be able to change in the future would be defined as a const at the top of the file. I guess it depends